src/Console/src/Usage/Model/OptionDefinition.php

```diff
@@ -43,18 +43,18 @@ class OptionDefinition
      */
     private $argument;
 
-    /**
-     * Option Definition
-     *
-     * @param string $shortName
-     * @param string $longName
-     * @param string $description
-     * @param string $type
-     * @param string $default
```

Comment:
`$default` could be `null|bool|string` currently. Which implies additional type checking every time the value is used. The "mixed" type is replaced by the `OptionValue` interface.